examples/call/scripts/localnet.sh

```
@@ -4,15 +4,15 @@ set -e
-if [ "$1" = "start" ]; then npx hardhat localnet & sleep 20; fi
+if [ "$1" = "start" ]; then npx hardhat localnet --skip solana & sleep 20; fi
 
 echo -e "\n🚀 Compiling contracts..."
 npx hardhat compile --force --quiet
 
 ZRC20_ETHEREUM=$(jq -r '.addresses[] | select(.type=="ZRC-20 ETH on 5") | .address' localnet.json)
 ERC20_ETHEREUM=$(jq -r '.addresses[] | select(.type=="ERC-20 USDC" and .chain=="ethereum") | .address' localnet.json)
 ZRC20_BNB=$(jq -r '.addresses[] | select(.type=="ZRC-20 BNB on 97") | .address' localnet.json)
-ZRC20_SOL=$(jq -r '.addresses[] | select(.type=="ZRC-20 SOL on 901") | .address' localnet.json)
+# ZRC20_SOL=$(jq -r '.addresses[] | select(.type=="ZRC-20 SOL on 901") | .address' localnet.json)
 ZRC20_SPL=$(jq -r '.addresses[] | select(.type=="ZRC-20 USDC on 901") | .address' localnet.json)
```

Shellcheck (0.10.0)

[warning] ZRC20_SPL appears unused. Verify use (or export if used externally).

(SC2034)